ntoskrnl/fsrtl/fastio.c

```diff
@@ -1605,7 +1605,11 @@ FsRtlAcquireFileExclusiveCommon(IN PFILE_OBJECT FileObject,
             FilterCallbacks->PostAcquireForSectionSynchronization(&CbData, Status, CompletionContext);
         }
 
-        return Status;
+        /* Return here only in specific status cases */
+        if (Status == STATUS_FILE_LOCKED_WITH_ONLY_READERS ||
+            Status == STATUS_FSFILTER_OP_COMPLETED_SUCCESSFULLY ||
+            Status == STATUS_FILE_LOCKED_WITH_WRITERS)
+            return Status;
```

Comment:
```suggestion
        {
            return Status;
        }
```

Reply:
OK, committed. But isn't it against our coding style? Can't check it on our Wiki because of 502.

Reply:
It makes sense to write it like suggested by Hermes, if the condition of the if-statement spans multiple lines. Even I do agree in this case, although I am typically a fan of vertical compactness and do typically omit the curly braces if neither the ifs condition nor the conditional command consumes more than a single line.